core/domain/takeout_service_test.py

```diff
@@ -320,6 +322,36 @@ class TakeoutServiceFullUserUnitTests(test_utils.GenericTestBase):
         {'cmd2': 'another_command'}
     ]
 
+    BASE_CLASSES = (
+        'BaseCommitLogEntryModel',
+        'BaseMapReduceBatchResultsModel',
+        'BaseModel',
+        'BaseSnapshotContentModel',
+        'BaseSnapshotMetadataModel',
+        'VersionedModel',
+    )
+
+    def _get_model_module_names(self):
+        """Get all module names in storage."""
+        # As models.NAMES is an enum, it cannot be iterated. So we use the
```

Comment:
iterated --> iterated over
or: "it is not iterable"

(here and in the next line)

Reply:
Done